hello-oboe/app/src/main/java/com/google/example/hellooboe/MainActivity.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.google.example.hellooboe
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.MotionEvent
+import android.view.View
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        val view = findViewById<View>(R.id.container)
+
+        // Set up a touch listener which calls the native sound engine
+        view.setOnTouchListener {_, event ->
+            if (event.action == MotionEvent.ACTION_DOWN) {
+                enableSound(true)
+            } else if (event.action == MotionEvent.ACTION_UP) {
+                enableSound(false)
+            } else {
+                return@setOnTouchListener false
+            }
+            true
+        }
+    }
+
+    override fun onResume() {
+        super.onResume()
+        createStream()
+    }
+
+    override fun onPause() {
+        destroyStream()
+        super.onPause()
+    }
+
+
+    // Creates and starts Oboe stream to play audio
+    private external fun createStream()
+
+    // Closes and destroys Oboe stream when app goes out of focus
+    private external fun destroyStream()
+
+    // Enables sound on user tap
+    private external fun enableSound(enable: Boolean)
```

Comment:
nit: change enableSound -> playSound, or just "play"?  just to make developers' life more enjoyable :-)
When you change, make sure the generated jni func name also changed

if possible, make this to take a return value so you know whether it is really executed ( if this is called before stream is created, you would return a false, for example )